src/views/abstractview.hpp

```diff
@@ -0,0 +1,28 @@
+#pragma once
+#include <QWidget>
+
+/**
+ * @class AbstractViewModel class
+ *
+ * An abstract base class that defines 2 basic functions that
```

Comment:
Forgot period at the end of sentence. (yeah ik very picky, but documentation should be complete sentences)